indico/modules/events/editing/fields.py

```diff
@@ -49,6 +50,13 @@ def validate_files(self, value):
                     raise ValidationError('File type "{}" does not allow these extensions: {}'
                                           .format(file_type.name, ', '.join(invalid_extensions)))
 
+            # ensure all filenames conform to the template
+            if file_type.filename_template:
+                filenames = {os.path.splitext(f.filename)[0] for f in files}
+                if not all(fnmatch.fnmatch(filename, file_type.filename_template) for filename in filenames):
+                    raise ValidationError("Some files don't conform to the filename template '{}'"
```

Comment:
unless we check the template locally (which I think would be nice), this needs to be translatable

Reply:
What do you mean by "locally" in this context?

Reply:
client-side (no need to waste people's time translating errors they never see unless they tinker with the data sent to the API)

Reply:
For now it is only backend validation (I will probably look into client-side validation once you are done with your PR)